modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -257,6 +257,65 @@ spec:
 * All result names were changed to uppercase.
 * The `gitInitImage` parameter was removed.
 
+[discrete]
+[id="op-taskref-jib-maven_{context}"]
+== jib-maven
+
+The `jib-maven` task builds Java, Kotlin, Groovy, and Scala sources into a container image by using the Jib tool for Maven projects. The Jib tool then pushes the image to the specified registry and provides an image digest for reference.
```

Comment:
I took this line from the yaml file, but I am not sure about the support downstream, @aneeshmbhat could you please let me know?? Thank you!